python/CMakeLists.txt

```diff
@@ -29,8 +30,8 @@ add_custom_target(wrapper ALL
 )
 
 if(NOT PYTHON_INSTALL_PARAMS)
-	set(PYTHON_INSTALL_PARAMS "--prefix=${CMAKE_INSTALL_PREFIX} --root=\$ENV{DESTDIR}/")
+	set(PYTHON_INSTALL_PARAMS "--no-index --find-links=dist/")  # TODO removing --prefix=${CMAKE_INSTALL_PREFIX} --root=\$ENV{DESTDIR}/ probably has downstream effects
```

Comment:
Note that I removed the arguments for --prefix and --root to simplify the Github Action test of the python module. I'm guessing this affects downstream scripts for publishing debian or nightly builds. I would recommend basing downstream scripts on the wheel file created in `cg3/python/dist/`, but another option is to expand python's search path in the GH Action to find the install.